scripts/run_alembic.sh

```
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+
+# Load environment variables from .env file
+if [ -f .env ]; then
+  export $(cat .env | sed 's/#.*//g' | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)